app/mailers/admin_mailer.rb

```diff
@@ -75,6 +75,7 @@ def auto_close_registrations
 
   def process_params
     @me = params[:recipient]
+    @user = @me.user
```

Comment:
As a side note on this ... I'm pretty sure the only reason we pass in `Account` instances and not `User` instances to the admin mailers is so that we can send an account along to the locale block wrapper.  If we change that to take user instead (which makes sense ... both locale and tz are kept on user) we can probably also change the admin mailer call locations to pass in a user instead of an account, and make those queries slightly smaller.

Reply:
The account is used to get the username in [admin_mailer.text.erb](https://github.com/mastodon/mastodon/blob/main/app/views/layouts/admin_mailer.text.erb).